near-sdk/src/test_utils/context.rs

```diff
@@ -29,10 +31,10 @@ impl VMContextBuilder {
     pub fn new() -> Self {
         Self {
             context: VMContext {
-                current_account_id: alice().into(),
-                signer_account_id: bob().into(),
+                current_account_id: alice().as_str().parse().unwrap(),
+                signer_account_id: bob().as_str().parse().unwrap(),
                 signer_account_pk: vec![0u8; 32],
-                predecessor_account_id: bob().into(),
+                predecessor_account_id: bob().as_str().parse().unwrap(),
```

Comment:
Wouldn't it make more sense to implement two-way `From`s for both `AccountId`s? Something like this:
```rust
impl From<near_vm_logic::types::AccountId> for AccountId {
    fn from(id: near_vm_logic::types::AccountId) -> Self {
        id.into()
    }
}

impl From<AccountId> for near_vm_logic::types::AccountId {
    fn from(id: AccountId) -> Self {
        id.into()
    }
}
```

Reply:
Because there is no direct conversion between the two. There is no ability to create a vm_logic AccountId without error checking. Can add a `TryFrom` impl for these, but that didn't seem like it would be worth it, given that it's just used internally and is basically the same amount of code.